examples/stable/test/property/ic_api/call/test/test.ts

```diff
@@ -1,5 +1,38 @@
 import { runTests } from 'azle/_internal/test';
+import { Actor, getCanisterId } from 'azle/_internal/test/actor';
 
 import { getTests } from './tests';
 
+export async function createActor(canisterName: string = 'call'): Promise<any> {
```

Comment:
I'm not sure why this is here, this code is automatically generated and should be obtained from the dfx_generated directory. There are many examples of how to do this in the examples directory.